docker/runtime/bash/http-trigger/startup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Since this shell script is so amazingly simple, I suggest using `/bin/sh` which might be a smaller/simpler/faster shell (like dash).  The only thing I can see below that is bash-specific is the use of `[[ ... ]]` (which in this case, can be simply replaced with vanilla posix `[ ... ]`)

Reply:
Ok, got it and changing.